llvm/test/CodeGen/AIE/aie2/schedule/postpipeliner/load-add-store-renamed.prepiplined.mir

```diff
@@ -18,21 +18,21 @@
   ; CHECK-NEXT:  // %bb.0: // %entry
   ; CHECK-NEXT:    mova r1, #0; nopb ; nopxm
   ; CHECK-NEXT:    ge r1, r1, r0
-  ; CHECK-NEXT:    jnz r1, #.LBB0_4
+  ; CHECK-NEXT:    jnz r1, #.LBB0_3
   ; CHECK-NEXT:    nop // Delay Slot 5
   ; CHECK-NEXT:    nop // Delay Slot 4
   ; CHECK-NEXT:    nop // Delay Slot 3
   ; CHECK-NEXT:    nop // Delay Slot 2
   ; CHECK-NEXT:    nop // Delay Slot 1
   ; CHECK-NEXT:  // %bb.1: // %for.body.preheader
-  ; CHECK-NEXT:    add.nc lc, r0, #-3
+  ; CHECK-NEXT:    add.nc lc, r0, #-1
```

Comment:
What happened here? I don't see any code change that could trigger this in this commit

Reply:
This is a refactor glitch. I rerun update_tests on all commits and will push shortly.